src/cluster_legacy.c

```diff
@@ -5986,6 +6023,7 @@ sds genClusterInfoString(void) {
 
     info = sdscatprintf(info,
                         "cluster_state:%s\r\n"
+                        "cluster_fail_reason:%s\r\n"
```

Comment:
I honestly would rather not add an info field. This is one of those things were if the cluster is down, you get more useful information from the other fields (cluster_slots_assigned and cluster_size) and I don't want to add *yet another field* to look at. It's also not necessarily one or the other, it could be both.

Reply:
yean, this does seem useless, @PingXie thoughts?

Reply:
agree, i removed it, please take a new review